packages/editor/src/browser/editor-decorations.ts

```diff
@@ -157,6 +83,36 @@ export interface DecorationOverviewRulerOptions {
     position?: OverviewRulerLane;
 }
 
+export enum OverviewRulerLane {
+    Left = 1,
+    Center = 2,
+    Right = 4,
+    Full = 7
+}
+
+export enum TrackedRangeStickiness {
+    AlwaysGrowsWhenTypingAtEdges = 0,
+    NeverGrowsWhenTypingAtEdges = 1,
+    GrowsOnlyWhenTypingBefore = 2,
+    GrowsOnlyWhenTypingAfter = 3,
+}
+
+export abstract class EditorDecorator {
```

Comment:
Should not it be `@injectable` as well? Could you double check seems like inversify dropped this requirement for super classes.

Reply:
Maybe move to another file.